types/params_test.go

```diff
@@ -288,6 +293,20 @@ func TestConsensusParamsUpdate(t *testing.T) {
 			},
 			updatedParams: makeParams(makeParamsArgs{evidenceAge: 3, precision: 2 * time.Second, messageDelay: 4 * time.Second}),
 		},
+		{
+			// update timeout params
+			intialParams: makeParams(makeParamsArgs{
+				abciExtensionHeight: 1,
+			}),
+			updates: &tmproto.ConsensusParams{
+				Abci: &tmproto.ABCIParams{
+					VoteExtensionsEnableHeight: 10,
```

Comment:
Would it make sense to test negative paths? (disabling, setting in the past, changing once initially set height is reached)

Reply:
Done.